### ISSUE TITLE
[Wunsch] Einzelne Räume ausblenden

### ISSUE THREAD

**Issue by Noxmiles** [EventFahrplan/EventFahrplan#100]
Da mit der Musik, den Assemblies und den selforganized Sessions mittlerweile sehr viele Spalten in der App sind wäre es sehr schön einzelne Spalten/Räume in den Einstellungen ausblenden zu können.

**Comment by torantal** [comment]
Being a couch participant this year for 35c3 it would be very convenient to have a "no stream/recording" filter feature to hide any entry that won't be transmitted (hoping this is an existing attribute). 

With this filter in effect all columns that are empty could then be automatically hidden.

**Comment by alexanderadam** [comment]
Wow, the no-stream addition is a splendid idea! 👍

**Comment by torantal** [comment]
It could of course work as a highlight as well for people at the conference "Go watch it because it won't be recorded!" :)

**Comment by steelman** [comment]
And/or rearrange the order of the columns. When I was searching for interesting sessions at FOSDEM2021 I found myself always swiping through a few tracks in the middle of the schedule. Having these columns at the end but not necessarily hidden would be nice too.

**Comment by erebion** [comment]
I like the idea. I would just like to see everything and quickly scroll through the whole day without changing rooms all the time, this way it just takes much longer.

I'd look just having a button like the "favourites" button that opens a list of all talk, so I can add those that I find interesting.

**Comment by johnjohndoe** [comment]
Thank you for your input @erebion. My impression from listening to the usage patterns of different people is that the app has the following usage scenarios which must be addressed differently:

1. Before the congress / before sessions take place: Browse the schedule to compose a list of favorites
2. At the congress / sessions are upcoming:  Have a good overview, see what is upcoming
3. After the congress / sessions took place: Revisit the schedule and favorites to find missed stuff or consume again

To address these usage patterns a well thought of design / UX flow has to be prepared.

**Comment by erebion** [comment]
Yup, but I think just seeing "Oh, that's next" without switching rooms a dozen times is the most common. I also think just being able to see everything in one column would help many users.

Also, @johnjohndoe, I think this would help with all thre use-cases you stated. :)
